src/v0/destinations/braze/transform.js

```diff
@@ -449,8 +451,46 @@ function processGroup(message, destination) {
   );
 }
 
-function process(event) {
+function processAlias(message, destination) {
   const respList = [];
+  const { aliasUpdates } =  constructPayload(message, mappingConfig[ConfigCategory.ALIAS.name]);
+  if (!aliasUpdates) {
+    throw new InstrumentationError('aliasUpdates key is required for alias call');
+  }
+
+  const aliasLabel = destination.Config.aliasLabel;
+
+  const perRequestMaxSize = 50;
+  for (let i = 0; i < aliasUpdates.length; i += perRequestMaxSize) {
+    const properChunk = aliasUpdates.slice(i, i + perRequestMaxSize);
+    const aliasUpdatesArray = [];
+
+    for (let obj of properChunk) {
+      let aliasObj = {};
+      // since both are required if not present ... skip them from final request
+      if (obj.oldAliasName && obj.newAliasName) {
+        aliasObj.old_alias_name = obj.oldAliasName;
+        aliasObj.new_alias_name = obj.newAliasName;
+        aliasObj.alias_label = obj.aliasLabel || aliasLabel;
+      }
+      aliasUpdatesArray.push(aliasObj);
+    }
+    const response = buildResponse(
+        message,
+        destination,
+        {
+          alias_updates: aliasUpdatesArray,
+        },
+        getAliasUpdateEndPoint(destination.Config.endPoint),
+    );
+    respList.push(response)
+  }
+
+  return respList;
+}
+
+function process(event) {
+  let respList = [];
```

Comment:
This doesn't seem correct. We will not get aliasUpdate array with our alias call.